src/main/java/com/tabnine/inline/CompletionPreviewUtils.kt

```diff
@@ -0,0 +1,7 @@
+package com.tabnine.inline
+
+import com.tabnine.prediction.TabNineCompletion
+
+fun hadSuffix(currentCompletion: TabNineCompletion): Boolean {
```

Comment:
typo - `hasSuffix`

Reply:
That is not a typo, as it refers to the oldSuffix